restyled Soldier Class Feats

bleehu opened the pull request:
and removed Battle Adrenaline, added Battle Hardened and Grizzled Veteran.
Simpler to play, and adds tanky-ness to the Soldier class in accordance
with design docs. Also removes another instance of old-instance nanites.

bleehu committed:
restyled Soldier Class Feats

and removed Battle Adrenaline, added Battle Hardened and Grizzled Veteran.
Simpler to play, and adds tanky-ness to the Soldier class in accordance
with design docs. Also removes another instance of old-instance nanites.

rosebrallier commented on CharacterCreation/02_Classes.txt:
```diff
@@ -44,40 +44,60 @@ Skill Proficiencies:
+
+    Your body and mind are used to the constant strain of combat. You no longer
+recoil quite so much from the horrors of war like a greenhorn and it takes more
+to shock you. Your pain tolerance has grown and you are more concious of your
```
sp: conscious.

rosebrallier commented on CharacterCreation/02_Classes.txt:
```diff
@@ -44,40 +44,60 @@ Skill Proficiencies:
+    You thrive on gunfighting. You love the smell of napalm in the morning. 
+You've had so much shrappnel in your body that it's gotten homeostatically used
+to it. This Feat cannot be taken unless you have "Battle Hardened" and are level 
+10. You gain the following.
```
"following:"

rosebrallier commented on CharacterCreation/02_Classes.txt:
```diff
@@ -44,40 +44,60 @@ Skill Proficiencies:
+    Your body and mind are used to the constant strain of combat. You no longer
+recoil quite so much from the horrors of war like a greenhorn and it takes more
+to shock you. Your pain tolerance has grown and you are more concious of your
+surroundings. You get the following:
```
"gain" (consistency)

Turtlelord26 commented on CharacterCreation/02_Classes.txt:
```diff
@@ -44,40 +44,60 @@ Skill Proficiencies:
+have beaten in the reflex roll when room clearing until that character takes a 
+turn.
+
+Surpise and Violence of Action:
```
sp: Surprise

Turtlelord26 reviewed (approved):
Remind me to merge these into my nanite rework superbranch.

rosebrallier commented on CharacterCreation/02_Classes.txt:
```diff
@@ -44,40 +44,60 @@ Skill Proficiencies:
+
+Raise Morale:
+
+	You roar to incite courage in your allies. 1 Action. 50% + (CHA mod) chance 
```
I think this is equivalent to making a DC 50 Cha check? If so, I think it's better to phase the ability in a way that is consistent with how we usually describe making rolls.

Turtlelord26 replied:
Confirm mathematically identical.

rosebrallier commented on CharacterCreation/02_Classes.txt:
```diff
@@ -44,40 +44,60 @@ Skill Proficiencies:
+Battle Cry:
+
+	You give a violent yell that spurs you and your teammates on, giving them
+the sense that victory is possible. DC 25 - CHA mod to be successful. If 
```
Same comment as above: DC 25 Cha check

rosebrallier commented on CharacterCreation/02_Classes.txt:
```diff
@@ -44,40 +44,60 @@ Skill Proficiencies:
+successful, your Battle Cry gives you and your teammates a +30 on their next 
+skill roll or +3 added to their next attack roll. A Character must be level 10 
+and have the "Raise Morale" Class Feat in order to take this feat. This ability
+takes 1 Action andcan only be used once per encounter. This ability replaces 
```
"and can"

rosebrallier commented on CharacterCreation/02_Classes.txt:
```diff
@@ -44,40 +44,60 @@ Skill Proficiencies:
+
+Fight Fiercely First:
+
+	You get an additional +10 to initiative rolls. In addition, you get a -3 
```
"gain"

rosebrallier commented on CharacterCreation/02_Classes.txt:
```diff
@@ -44,40 +44,60 @@ Skill Proficiencies:
+Fight Fiercely First:
+
+	You get an additional +10 to initiative rolls. In addition, you get a -3 
+miss chance against enemies who have not taken a turn yet in combat or whom you
```
"+3 accuracy"

rosebrallier commented on CharacterCreation/02_Classes.txt:
```diff
@@ -44,40 +44,60 @@ Skill Proficiencies:
+	The soldier's accuracy bonus effect works on any enemy with a lower 
+initiative until that enemy has finished their 3rd turn. A Character cannot take
+this Class Feat unless they are level 10 and have the "Fight Fiercely First"
+Class Feat as well.
```
delete "as well."

rosebrallier reviewed (changes requested):
XD
I know you didn't set out to change Raise Morale much, but I have suggestions (and concerns about how the original was written)

bleehu commented:
Dorp. My bad.

bleehu commented:
Ya, I've got some hesitation about it too. I think it originated as a way to try to force charisma to have combat relevance. Could you capture your concerns either here, in CX design, or on slack? Whichever works best for you.

rosebrallier commented:
My concerns are more from a "writing a comprehensible thing" perspective than a gameplay perspective.

bleehu committed:
spelling and grammar fixes
for Soldier class feats

rosebrallier commented on CharacterCreation/02_Classes.txt:
```diff
@@ -44,40 +44,61 @@ Skill Proficiencies:
+10. You gain the following:
+
+* An additional +20 to your maximum health
+* If you would fail a reload roll, you succeed instead    
```
Maybe we should exclude crit fails from this?

rosebrallier reviewed (approved):
Ship-it! I left one suggestion re allowing failure, but take it or leave it as you see fit.

bleehu committed:
added addendum to Grizzled Vetran

As per RoseB's request, we've excluded crit fails from being overridden by the Grizzled veteran feat.
